conformance/conformance.proto

```diff
@@ -57,11 +57,13 @@ option java_package = "com.google.protobuf.conformance";
 //   2. parse the protobuf or JSON payload in "payload" (which may fail)
 //   3. if the parse succeeded, serialize the message in the requested format.
 message ConformanceRequest {
+  string test_name = 1;
```

Comment:
I don't expect the testee to use the name programmatically, but it is useful to have around for debugging messages. It's nice that when you log the request, you can also see the test name. Right now the runner just logs the entire request message when there is a failure.